### PR TITLE
fix: update Container monitoring add-on with 11012019 agent image changes

### PR DESCRIPTION
**Pull request by ganga1980**
**Reason for Change**:

**Issue Fixed**:

This PR has changes related updates for the Container monitoring add-on with latest agent image version: 11012019

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: parts/k8s/containeraddons/1.16/kubernetesmasteraddons-omsagent-daemonset.yaml [hunk 1]
```diff
@@ -33,7 +33,7 @@ rules:
   resources: ["pods", "events", "nodes", "namespaces", "services"]
   verbs: ["list", "get", "watch"]
 - apiGroups: ["extensions"]
-  resources: ["deployments"]
+  resources: ["replicasets"]
```
Comment (devigned) [inline parts/k8s/containeraddons/1.16/kubernetesmasteraddons-omsagent-daemonset.yaml]: Why change to replicasets over deployments?

Comment (ganga1980) [reply parts/k8s/containeraddons/1.16/kubernetesmasteraddons-omsagent-daemonset.yaml]: This is for our health service to build model for the workloads. This work has been done by other team member. Based on the discussion with him,  we can get metadata.ownerReferences  which used to build hierarchy  from bottom up and also handle the deployment version updates from v1 to v2 etc.

**Comment by devigned** [comment]
@ganga1980 are you able to run the tests locally in the dev container? I believe this will help to avoid the test failures in the CI system.

**Comment by ganga1980** [comment]
> @ganga1980 are you able to run the tests locally in the dev container? I believe this will help to avoid the test failures in the CI system.
> @devigned , I ran the tests locally. all the tests are getting passed. In-fact, I have ran "make ensure-generated"  and also ran the test before pushing any code.  Not sure why CI failing with mismatch in generated code.

**Comment by devigned** [comment]
@ganga1980 I would run `make reload` from the tools directory to ensure you have the correct version of tools installed. see: https://github.com/Azure/aks-engine/blob/master/hack/tools/Makefile#L22

**Comment by ganga1980** [comment]
> @ganga1980 I would run `make reload` from the tools directory to ensure you have the correct version of tools installed. see: https://github.com/Azure/aks-engine/blob/master/hack/tools/Makefile#L22

@devigned, Thanks for pointing this. I was knocking my head why the generated files difference in CI not locally. I have ran "make reload", "make generate" and "make test", and tested my changes by creating cluster with addon. I think, this time would make the CI happy.

**Comment by ganga1980** [comment]
> > @ganga1980 I would run `make reload` from the tools directory to ensure you have the correct version of tools installed. see: https://github.com/Azure/aks-engine/blob/master/hack/tools/Makefile#L22
> 
> @devigned, Thanks for pointing this. I was knocking my head why the generated files difference in CI not locally. I have ran "make reload", "make generate" and "make test", and tested my changes by creating cluster with addon. I think, this time would make the CI happy.

@devigned , seems all the tests and checks are passed. when you get a chance, please help on merging this PR?

**Comment by ganga1980** [comment]
Hi, @devigned , When you get a chance, can you please help on merging this PR?

**Pull request merged**